src/join_mesh.py

```diff
@@ -5,16 +5,6 @@
 There are two possible ways of joining the meshes:
 - Partition-wise (does not recover the original mesh order lacks of discarded cells)
 - Recovery merge (maintain original mesh order and adds discarded cells)
-
-Sample usage
-
-partitioned files:
-prefix_<#>.vtu
-
-./join_mesh.py -m prefix -o joined_mesh.vtk -dir joined_mesh_dir
-
-./join_mesh.py -m prefix -o joined_mesh.vtk -r ./path/to/recovery.json
-
```

Comment:
Unrelated

Reply:
Same below.

Reply:
Ok then let's put at least a sample usage of all required arguments in the --help message.